exporting/prometheus/remote_write/remote_write.c

```diff
@@ -177,6 +177,30 @@ int format_host_prometheus_remote_write(struct instance *instance, RRDHOST *host
     add_host_info(
         connector_specific_data->write_request,
         "netdata_info", hostname, host->program_name, host->program_version, now_realtime_usec() / USEC_PER_MS);
+    
+    if (unlikely(sending_labels_configured(instance))) {
+        struct format_remote_write_label_callback tmp = {
+            .write_request = connector_specific_data->write_request,
+            .instance = instance
+        };
+        rrdlabels_walkthrough_read(host->host_labels, format_remote_write_label_callback, &tmp);
+    }
+
+    add_host_info(
+        connector_specific_data->write_request,
+        "netdata_host_tags", hostname, host->program_name, host->program_version, now_realtime_usec() / USEC_PER_MS);
+    
+    if (unlikely(sending_labels_configured(instance))) {
+        struct format_remote_write_label_callback tmp = {
+            .write_request = connector_specific_data->write_request,
+            .instance = instance
+        };
+        rrdlabels_walkthrough_read(host->host_labels, format_remote_write_label_callback, &tmp);
+    }
+
+    add_host_info(
+        connector_specific_data->write_request,
+        "netdata_host_tags_info", hostname, host->program_name, host->program_version, now_realtime_usec() / USEC_PER_MS);
```

Comment:
Why do we need this part? I think it should be removed.

Reply:
Well, we have `netdata_host_tags` and `netdata_host_tags_info`. I don't know for what reason though 🤷‍♂️.

Reply:
Historically, we had `host tags` option in the `[backends]` section. The backends subsystem was removed completely. I don't see any point in duplicating the same labels in three places. I think that we shouldn't add `netdata_host_tags` and `netdata_host_tags_info` to the remote write connector.

Reply:
I would like to keep same as API see, or it sometimes let me feel very tricky that why some thing not send. Or we should remove `netdata_host_tags` and `netdata_host_tags_info` in API output

Reply:
Then I prefer to remove them from the API output.

Reply:
May be some one is using metric...

Reply:
We announced breaking changes when we deleted the backends subsystem. There is no need to keep these metrics. If you wish, we'll delete the metrics from the API, just remove them from this PR.

Reply:
fine, i will change the PR